src/Entity/ConsularDistrict.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\ConsularDistrictRepository")
+ * @ORM\Table(
+ *     name="consular_districts",
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="consular_district_code_unique", columns="code"),
+ *         @ORM\UniqueConstraint(name="consular_district_referent_tag_unique", columns="referent_tag_id")
+ *     }
+ * )
+ */
+class ConsularDistrict
```

Comment:
This entity should represent the table of consular districts (little different from existing `District` entity):
![image](https://user-images.githubusercontent.com/4582866/72921009-bca49b80-3d4a-11ea-89e6-fbcb1df7e559.png)
which has `id`, `countries`, `cities` and `code` columns.
Could you update this entity for matching the source of data?

Reply:
Isn't the "N° circo" column the number, and the code a different thing?